src/PowerPlaywright/LocalAssemblyProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace PowerPlaywright
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+
+    /// <summary>
+    /// Retrieves a control assembly from the test assembly directory.
+    /// </summary>
+    internal class LocalAssemblyProvider : IAssemblyProvider
+    {
+        private readonly string path;
+
+        private Assembly assembly;
+
+        public LocalAssemblyProvider(string path)
+        {
+            this.path = path ?? throw new ArgumentNullException(nameof(path));
+        }
+
+        /// <inheritdoc />
+        public Assembly GetAssembly()
+        {
+            if (this.assembly == null)
+            {
+                var assemblyPath = Path.Combine(Path.GetDirectoryName(typeof(LocalAssemblyProvider).Assembly.Location), this.path);
+
+                this.assembly = Assembly.LoadFrom(assemblyPath);
```

Comment:
I cannot be 100% sure but it think this member changed to Assembly.Load(assemblyString), I will pull this branch locally and check.

Reply:
Nope its correct Assembly.Load does not work. You are correct here @ewingjm